app/services/analytics_events.rb

```diff
@@ -1611,6 +1611,24 @@ def idv_in_person_switch_back_visited(flow_path:, **extra)
     track_event('IdV: in person proofing switch_back visited', flow_path: flow_path, **extra)
   end
 
+  # An email from USPS with an enrollment code has been received, indicating
+  # the enrollment is approved or failed. A check is required to get the status
+  # it is not included in the email.
+  # @param [boolean] multi_part If the email is marked as multi_part
+  # @param [string] part_found Records if the enrollment code was found in text_part or html_part
+  def idv_in_person_usps_proofing_enrollment_code_email_received(
+    multi_part: nil,
+    part_found: nil,
+    **extra
+  )
+    track_event(
+      'IdV: in person usps proofing enrollment code email received',
+      multi_part: multi_part,
+      part_found: part_found,
+      **extra,
+    )
+  end
```

Comment:
What do you think about adding a test that checks event logs for analytics attributes?

Reply:
Oh yeah, that's probably a good idea. 👍